app/mailers/application_mailer.rb

```diff
@@ -14,8 +14,8 @@ class ApplicationMailer < ActionMailer::Base
     template_path: ->(mailer) { "mailers/#{mailer.class.name.underscore}" },
   )
 
-  def email_from(topic)
-    "#{SiteConfig.community_name} #{topic} <#{SiteConfig.email_addresses[:default]}>"
+  def email_from(_)
+    "#{SiteConfig.community_name} <#{SiteConfig.email_addresses[:default]}>"
```

Comment:
I removed `#{topic}` per your suggestion, but was unable to remove the `topic` argument without running into some weird failures and this seemed to do the trick! cc @vaidehijoshi

Reply:
Did you remove the “Community” argument from where we invoke the email_from method as well?

https://github.com/forem/forem/blob/8f191d7fa4f1226aa57256b9b5b59b12b627aa8e/app/mailers/application_mailer.rb#L13

I tried it locally and removed it from both places (and didn’t run into any failures)

Reply:
Giant 🤦‍♀️ ...

Reply:
It happens to the best of us! 🙈

And it's been a heck of a long year 😅